app/models/ext_management_system.rb

```diff
@@ -69,6 +69,7 @@ def self.api_allowed_attributes
   has_one  :iso_datastore, :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
 
   belongs_to :zone
+  belongs_to :backup_zone, :class_name => "Zone", :foreign_key => "backup_zone_id", :inverse_of => :paused_ext_management_systems # used for maintenance mode
```

Comment:
Do you need the `foreign_key` option...I would have expected that to be automatic?